src/content/docs/curriculum-help.mdx

```diff
@@ -197,6 +197,25 @@ To check the return value of `__str__` you would write:
 });
 ```
 
+AST-based helpers are also exposed in the Python challenges in `ast_helpers` module. `_Node` class corresponds to the `ast_helpers.Node`:
```

Comment:
you still need to run the python code through pyodide, right?

Reply:
Uhm, not by yourself. What I'm trying to communicate is this:

1. Go to python challenge, ie.: https://www.freecodecamp.org/learn/scientific-computing-with-python/learn-how-to-work-with-numbers-and-strings-by-implementing-the-luhn-algorithm/step-15
2. import `ast_helpers`
3. Use `ast_helpers.Node` to parse code and play around with various methods of the class.

Reply:
you mean to use the ast_helpers in the editor?

Reply:
Yeah